x-pack/plugins/security/public/views/management/edit_user.html

```diff
@@ -1,184 +1,3 @@
 <kbn-management-app section="security" omit-breadcrumb-pages="['edit']">
-  <div class="kuiViewContent kuiViewContent--constrainedWidth kuiViewContentItem">
-    <!-- Subheader -->
-    <div class="kuiBar kuiVerticalRhythm">
-      <div class="kuiBarSection">
-        <!-- Title -->
-        <h1 class="kuiTitle">
-          <span ng-if="editUser.isNewUser">
-            Add user
-          </span>
-          <span ng-if="!editUser.isNewUser">
-            &ldquo;{{ user.username }}&rdquo; User
-          </span>
-        </h1>
-      </div>
-
-      <div class="kuiBarSection">
-        <!-- Delete button -->
-        <button
-          ng-if="!editUser.isNewUser && !user.metadata._reserved"
-          class="kuiButton kuiButton--danger kuiButton--iconText"
-          ng-click="deleteUser(user)"
-          tooltip="Delete User"
-          data-test-subj="userFormDeleteButton"
-        >
-          <span class="kuiButton__inner">
-            <span class="kuiButton__icon kuiIcon fa-trash"></span>
-            <span>Delete user</span>
-          </span>
-        </button>
-        <div
-          ng-if="user.metadata._reserved"
-          class="kuiBadge kuiBadge--default"
-          tooltip="Reserved users are built-in and cannot be removed. Only the password can be changed."
-        >
-          <span class="kuiIcon fa-lock"></span>
-          Reserved
-        </div>
-      </div>
-    </div>
-
-    <!-- Form -->
-    <form name="form" novalidate class="kuiVerticalRhythm">
-      <!-- Username -->
-      <div class="kuiFormSection">
-        <label for="username" class="kuiFormLabel">
-          Username
-        </label>
-        <input
-          type="text"
-          ng-class="::editUser.isNewUser ? 'kuiTextInput fullWidth' : 'kuiStaticInput'"
-          ng-disabled="!editUser.isNewUser"
-          id="username"
-          name="username"
-          ng-model="user.username"
-          required pattern="[a-zA-Z_][a-zA-Z0-9_@\-\$\.]*"
```

Comment:
I think we lost this validation -- I can create a user with a username consisting of just special characters. It looks like ES (or maybe Kibana itself) strips out some of those characters, so the username I typed isn't the username that was actually saved.

Reply:
Good catch, added that back in.